### ISSUE TITLE
[Bug] BE automatically disconnects after restarting

### ISSUE THREAD

**Issue by Weirdolin** [apache/doris#25590]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Version

doris-2.0-beta-afe6bb9

### What's Wrong?

When I restart a BE, it automatically disconnects after a few seconds of successful connection.
Other BE nodes are normal. I tried to upgrade this failed BE to version 2.0.2, but this error still occurs.
An error appears in the database:
![2fdf52c4-9006-46e8-9509-37a09f64a41f](https://github.com/apache/doris/assets/85385001/a40edbaa-4e2d-44a0-8913-65f398413f0a)
java.net.ConnectException: Connection refused (Connection refused)
An error appears in the BE(be.WARNING):
![img_v2_cfac359e-6e51-48df-9d7a-ec25f956f9fg](https://github.com/apache/doris/assets/85385001/643eaf02-1754-4f68-aa0e-149c14a75afd)
failed to serialize meta 136209.1010901.9e4d5b428d0afa25-9f2a64c5771113a84

### What You Expected?

BE node can start normally
I very much hope to get your help, thank you

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by LemonLiTree** [comment]
Check it out under the 2.0.2 test, and also confirm whether be starts normally? `jps`

**Comment by Weirdolin** [comment]
> Check it out under the 2.0.2 test, and also confirm whether be starts normally? `jps`

Yes, I updated this error BE to 2.0.2, there is no change, how many seconds after normal startup will report this

**Comment by LemonLiTree** [comment]
how to configure `priorty_network` in be.conf and fe.conf

**Comment by Weirdolin** [comment]
> how to configure `priorty_network` in be.conf and fe.conf

FE:priority_networks = 10.96.3.0/24
BE:priority_networks = 10.208.0.73/32
I now want to use "ALTER SYSTEM DECOMMISSION BACKEND" to achieve safe removal of BE，However, no decrease in TabletNum was found。

**Comment by LemonLiTree** [comment]
add my wechat ：LQF1296996755

**Comment by zhangyuxiansen2017** [comment]
相同问题，重启be报：Access denied for user 'root@10.244.0.60' (using password: NO)
